infra/roles/products/files/apache_vhost

```diff
@@ -0,0 +1,6 @@
+<VirtualHost *:80>
+
+    ServerName product.arcada.nitor.zone
+
+    ProxyPass "/"  "http://localhost:8080/"
```

Comment:
This needs to be different for PHP, you cannot proxy, but instead have to serve an actual directory